config/jobs/kubernetes/sig-testing/compatibility-versions-e2e.yaml

```diff
@@ -165,3 +165,61 @@ periodics:
           # this is mostly for building kubernetes
           memory: 14Gi
           cpu: 7
+- interval: 1h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-kind-two-steps-upgrade-n-minus-1
+  annotations:
+    testgrid-dashboards: sig-testing-kind
```

Comment:
Not a blocker for this CL, but please consider it for a future update: cc @siyuanfoundation @aaron-prindle 

I don't think this makes sense for SIG Testing to own at this point, this is primarily testing a SIG API Machinery feature, and not KIND itself.

Let's move these to a SIG API Machinery folder and dashboard (not urgent, but let's do that before it gets a lot of inbound links)